### PR TITLE
[performance](load) increase max_broker_concurrency to 100

### PR DESCRIPTION
**Pull request by kaijchen**
### What problem does this PR solve?

Increase default `max_broker_concurrency` to 100 to improve broker load performance.
This option will affect the max number of scan / sink instances allowed in a broker load.

```
parallel instance = min(max_broker_concurrency,
                        source file size / min_bytes_per_broker_scanner,
                        num backends * load_parallelism)
```

S3 load time of tpcds_1000g catalog_sales:
* before: 438s
* after: 225s

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [x] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [x] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [x] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -780,7 +780,7 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true, masterOnly = true, description = {
             "单个 broker scanner 的最大并发数。", "Maximal concurrency of broker scanners."})
-    public static int max_broker_concurrency = 10;
+    public static int max_broker_concurrency = 24;
```
Comment (kaijchen) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:783]:
```suggestion
    public static int max_broker_concurrency = 100;
```

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**